webhooks-extension/docs/Labels.md

```diff
@@ -0,0 +1,64 @@
+# Labelling PipelineRuns
+
+The webhooks extension's graphical user interface (GUI) provides the ability to display the status of the latest pipelinerun for branches of a specified webhook.  This listing is seen by clicking on the webhook name in the webhook table.
+
+For this functionality to work, users must:  
+<br/>
+
+1. Add labels to the pipelineruns metadata defined in the pipeline's triggertemplate. Three of the values associated with the labels are automatically made available to the triggertemplate by the webhooks extension.  The labels required are:
```

Comment:
Could this be restructured to make it clearer that, 
- You must add four labels
- Three are made available automatically
- git-branch needs extra help from the triggerBinding because it's event-specific

Reply:
Have a look now and see what you think